fixtures/components/complex-types/table/index.tsx

```diff
@@ -3,6 +3,8 @@
 import * as React from 'react';
 import { TableProps } from './interfaces';
 
+export { TableProps };
```

Comment:
The new implementation is more strict, it checks that the interface is actually exported in the index file (which was our requirement all the time)